LunarVim/LunarVim
# [Feature] Configurable color highlights

# Description

I added an additional configuration to override color highlights. I also refactored the color folder so it will be more convenient to manage. But since it's a feature, I put the [Feature] handler instead of [Refactor], maintainers feel free to edit the PR title. Config snippet:

```lua
-- ...
O.colors.colorscheme = "spacegray"
O.colors.highlights = {
  Visual = { bg = "#ffffff" },
  VimWikiHeader1 = {  fg="#dddddd", bg="#111111" },
}
-- ...
```

Fixes #1062 

## How Has This Been Tested?

- Change the `lv-config.lua`, add some highlight config
- Reopen lvim, with command `lvim -V9lvimLog.log`
- Check logs `lvimLog.log` (no issue)
